static/app/components/menuListItem.tsx

```diff
@@ -105,8 +106,19 @@ function BaseMenuListItem({
   forwardRef,
   ...props
 }: Props) {
+  const labelId = useMemo(() => domId('menuitem-label-'), []);
+  const detailId = useMemo(() => domId('menuitem-details-'), []);
+
   return (
-    <MenuItemWrap as={as} ref={forwardRef} {...props}>
+    <MenuItemWrap
+      role="menuitem"
```

Comment:
This will be overriden in `SelectOption`, but this can be considered the default role